airflow/contrib/operators/jenkins_job_trigger_operator.py

```diff
@@ -35,27 +35,21 @@
     basestring = str  # For python3 compatibility
 
 
-# TODO Use jenkins_urlopen instead when it will be available
-# in the stable python-jenkins version (> 0.4.15)
```

Comment:
Do you know what this comment was about? Do we need to update the version we specify in setup.py for `jenkins_request()` method to be available?

Reply:
I think this was added in 1.0.0.

Reply:
Seems that jenkins_request was added in python-jenkins 1.0.0 
That is also the version where they changed the authentication mechanism that causes original operator not to work. Authentication no more initialized on connection creation, there is an internal method for that now.
1.0.0 - https://git.openstack.org/cgit/openstack/python-jenkins/tree/jenkins/__init__.py?h=1.0.0
0.4.16 (single release between 0.4.15 and 1.0.0) - https://git.openstack.org/cgit/openstack/python-jenkins/tree/jenkins/__init__.py?h=0.4.16

Reply:
Added a commit to update setup.py